pkg/bid/bid.go

```diff
@@ -0,0 +1,113 @@
+package bid
+
+import (
+	"fmt"
+	"strings"
+
+	v2 "github.com/conductorone/baton-sdk/pb/c1/connector/v2"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+const (
+	BidPrefix            = "bid"
+	ResourceBidPrefix    = "r"
+	EntitlementBidPrefix = "e"
+	GrantBidPrefix       = "g"
+)
+
+type BID interface {
```

Comment:
should we inline a bunch of the docs from the wiki proposal here detailing the format?  either here or as a package level doc string?

Reply:
I added a docs to the top of this file. I think it's the correct format for go docs.